### PR TITLE
Custom InfoCard background

### PR DESCRIPTION
**Pull request by Mintnoii**

### CODE REVIEW

File: docs/.vuepress/config.js [hunk 1]
```diff
@@ -55,7 +55,7 @@ module.exports = {
       organization: 'Xi\'an Jiao Tong University',
 
       avatar: 'https://www.meteorlxy.cn/assets/img/avatar.jpg',
-
+      background: 'https://www.meteorlxy.cn/assets/img/avatar.jpg',
```
Comment (meteorlxy) [inline docs/.vuepress/config.js]: Empty line between different properties is preferred

File: lib/index.js [hunk 2]
```diff
@@ -1,5 +1,4 @@
 const path = require('path')
-
```
Comment (meteorlxy) [inline lib/index.js]: This should not be included

File: docs/_posts/2019-02-26-theme-guide-zh.md [hunk 3]
```diff
@@ -134,7 +134,13 @@ module.exports = {
...
       // 或者放置在 .vuepress/public 文件夹，例如 .vuepress/public/img/avatar.jpg
       // avatar: '/img/avatar.jpg',
-      
+
+      // 个人信息背景图片
+      background: 'https://www.meteorlxy.cn/assets/img/avatar.jpg',
+      // 使用外链或者放置在 .vuepress/public 文件夹，例如 .vuepress/public/img/background.jpg
+      // background: '/img/background.jpg',
+      // 如果不设置会随机生成一个图案
+
```
Comment (meteorlxy) [inline docs/_posts/2019-02-26-theme-guide-zh.md]: the english docs should be updated, too

**Review by meteorlxy — CHANGES_REQUESTED** [review]
What's more, if the header background and the infocard background do not use geo in the mean time, we should find a way to remove geopattern to avoid it be bundled in.

File: lib/index.js (hunk 2 above)
Comment (Mintnoii) [reply lib/index.js]: sorry, my bad.

**Comment by Mintnoii** [comment]
> What's more, if the header background and the infocard background do not use geo in the mean time, we should find a way to remove geopattern to avoid it be bundled in.

About this, do you have any good ideas? I don't know if we can use `import()` to dynamically load `Geopattern`.And this method seems to need to configure `Syntax Dynamic Import Babel Plugin` in `chainWebpack`.

**Comment by meteorlxy** [comment]
One possible way is to use `define` (webpack alias) to point `geopattern` to an empty file.

By default, `import()` is supported by Vuepress, but I'm afraid it's not a valid solution for this problem

**Pull request closed without merging**